spec/decorators/comment_decorator_spec.rb

```diff
@@ -1,26 +1,42 @@
 require "rails_helper"
 
 RSpec.describe CommentDecorator, type: :decorator do
+  context "with serialization" do
+    let_it_be_readonly(:comment) { create(:comment).decorate }
+
+    it "serializes both the decorated object IDs and decorated methods" do
+      expected_result = { "id" => comment.id, "published_timestamp" => comment.published_timestamp }
+      expect(comment.as_json(only: [:id], methods: [:published_timestamp])).to eq(expected_result)
```

Comment:
Do we test serializing all of the attribute methods `as_json` anywhere?

Reply:
Not all the attributes, I replicated most examples using the various parameters. We do test the serialization of the homepage here: https://github.com/thepracticaldev/dev.to/pull/6125/files#diff-786f309cff05f0950d73a2b25f188fb1R79

Reply:
Oh nice bc if serializing a field was broken that would return a 500 👍